### ISSUE TITLE
pull-test-infra-bazel flakes upon installing gubernator requirements.txt

### ISSUE THREAD

**Issue by krzyzacy** [kubernetes/test-infra#4989]
sample: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/test-infra/4986/pull-test-infra-bazel/8659/

```
I1013 00:25:37.785] Call:  /workspace/./test-infra/jenkins/../scenarios/kubernetes_bazel.py --build=//... --install=gubernator/test_requirements.txt '--test=//... //verify:verify-all' --test-args=--test_output=errors
W1013 00:25:37.861] Run: ('pip', 'install', '-r', 'gubernator/test_requirements.txt')
I1013 00:25:39.576] Collecting webtest (from -r gubernator/test_requirements.txt (line 1))
I1013 00:25:39.807]   Downloading WebTest-2.0.28-py2.py3-none-any.whl
I1013 00:25:39.882] Collecting nosegae (from -r gubernator/test_requirements.txt (line 2))
I1013 00:25:39.949]   Downloading NoseGAE-0.5.10.tar.gz
I1013 00:25:40.026]     Complete output from command python setup.py egg_info:
I1013 00:25:40.027]     Traceback (most recent call last):
I1013 00:25:40.027]       File "<string>", line 1, in <module>
I1013 00:25:40.027]     ImportError: No module named setuptools
```

**Comment by BenTheElder** [comment]
`setuptools` isn't being installed, which is causing WebTest to fail, because our python deps are broken: https://github.com/kubernetes/test-infra/issues/4849

**Comment by xiangpengzhao** [comment]
also see in https://github.com/kubernetes/test-infra/pull/4991

**Comment by BenTheElder** [comment]
We probably need to move setuptools to requirements.txt or to being installed in the image now instead of in here https://github.com/kubernetes/test-infra/blob/master/gubernator/test_requirements.txt

**Comment by BenTheElder** [comment]
this looks to be fine now. fixed in #4997 and #4998
/close
